packages/typespec-autorest-canonical/README.md

```diff
@@ -0,0 +1,78 @@
+# @azure-tools/typespec-autorest-canonical
+
+TypeSpec library for emitting canonical swagger
+
+## Install
+
+```bash
+npm install @azure-tools/typespec-autorest-canonical
+```
+
+## Emitter
+
+### Usage
+
+1. Via the command line
+
+```bash
+tsp compile . --emit=@azure-tools/typespec-autorest-canonical
+```
+
+2. Via the config
+
+```yaml
+emit:
+  - "@azure-tools/typespec-autorest-canonical"
+```
+
+### Emitter options
+
+#### `output-file`
+
+**Type:** `string`
+
+Name of the output file.
+Output file will interpolate the following values:
+
+- service-name: Name of the service if multiple
+- azure-resource-provider-folder: Value of the azure-resource-provider-folder option
+
+Default: `{azure-resource-provider-folder}/{service-name}/{version}/openapi.json`
+
+Example: Single service
+
+- `canonical.openapi.json`
+
+Example: Multiple services
+
+- `Service1.canonical.openapi.json`
+- `Service2.canonical.openapi.json`
+
+#### `azure-resource-provider-folder`
+
+**Type:** `string`
+
+#### `arm-types-dir`
```

Comment:
I don' tthink you need `arm-types-dir` if you are not going to interpolate the ref set. (there is also a bit of cleanup you might be able to do in `opnapi.ts`

Reply:
removed